katas/content/complex_arithmetic/polar_multiplication/Verification.qs

```diff
@@ -1,27 +1,26 @@
-namespace Kata.Verification {        
-    open Microsoft.Quantum.Convert;
-    open Microsoft.Quantum.Math;
+namespace Kata.Verification {
+    import Std.Convert.*;
+    import Std.Math.*;
 
     @EntryPoint()
     operation CheckSolution() : Bool {
-        for _ in 0 .. 24 {
+        for _ in 0..24 {
             let x = DrawRandomComplex();
             let y = DrawRandomComplex();
             let xp = ComplexAsComplexPolar(x);
             let yp = ComplexAsComplexPolar(y);
 
             let expected = ComplexAsComplexPolar(TimesC(x, y));
             let actual = Kata.ComplexPolarMult(xp, yp);
-        
+
             if not ComplexPolarEqual(expected, actual) {
                 Message("Incorrect");
-                Message($"For x = {ComplexPolarAsString(xp)}, y = {ComplexPolarAsString(yp)} " + 
-                    $"expected return {ComplexPolarAsString(expected)}, actual return {ComplexPolarAsString(actual)}.");
+                Message($"For x = {ComplexPolarAsString(xp)}, y = {ComplexPolarAsString(yp)} " + $"expected return {ComplexPolarAsString(expected)}, actual return {ComplexPolarAsString(actual)}.");
```

Comment:
Is this our recommendation now - not splitting long expression to the next line? If so, we can just drop the + and make it one string. But I like it better the way it was.

Reply:
I've gone through and manually fixed some of the formatting issues. Most of which were just a matter of adding a newline and then the formatter picks a more appropriate format, but for these messages that get broken with a `+` I had to forego the formatter unfortunately.